api/groups/nodeGroup_test.go

```diff
@@ -125,6 +126,66 @@ func TestHeartbeatstatus(t *testing.T) {
 	assert.NotEqual(t, "", statusRsp.Message)
 }
 
+func TestP2PMetrics_ShouldReturnErrorIfFacadeReturnsError(t *testing.T) {
```

Comment:
perhaps a new `TestQueryDebug_ShouldReturnErrorIfFacadeReturnsError` for extra coverage

Reply:
the situation is already covered in `TestQueryDebug_GetQueryErrorsShouldErr`